ngraph/python/tests/test_ngraph/test_ops_unary.py

```diff
@@ -100,14 +99,13 @@ def sigmoid(x):
     assert np.allclose(result, expected)
 
 
-@xfail_issue_44970
 def test_softmax():
-    axis = 0
+    axis = 1
     input_tensor = np.array([[1, 2, 3], [4, 5, 6]], dtype=np.float32)
 
     result = run_op_node([input_tensor], ng.softmax, axis)
 
-    expected = [[0.00426978, 0.01160646, 0.03154963], [0.08576079, 0.23312202, 0.6336913]]
```

Comment:
it is a different test now, I wonder if it can be called a fix for issue.
Why did you change axis and results?

Reply:
The previous results were incorrect for `axis == 0`. The correct values are:
```
[[0.04742587, 0.04742587, 0.04742587],
 [0.95257413, 0.95257413, 0.95257413]]
```
I've changed both the axis and correct expected values in this test.